webui/react/src/pages/FlatRuns/FlatRuns.tsx

```diff
@@ -1029,7 +1013,7 @@ const FlatRuns: React.FC<Props> = ({ projectId, searchId }) => {
               getHeaderMenuItems={getHeaderMenuItems}
               getRowAccentColor={getRowAccentColor}
               imperativeRef={dataGridRef}
-              isPaginated={isPagedView}
+              isPaginated
```

Comment:
i know its from hew, but do we wanna remove that or leave it for the potential use case?

Reply:
good catch, I think instead of removing the functionality entirely from `DataGrid`, i'll replace `isPaginated` with an optional `infiniteScroll` prop that is false by default. maybe should've implemented it that way in the first place. will make a hew PR and then update this one when that's merged.